terraform/modules/bosh_vpc/network_private.tf

```diff
@@ -135,3 +135,26 @@ resource "aws_instance" "az2_private_nat_2017_09" {
     Name = "${var.stack_description}  (AZ2 2017-09 NAT)"
   }
 }
+
+# AWS NAT Gateway provided by AWS
+resource "aws_nat_gateway" "az1_private_nat_gateway" {
+  allocation_id = "${aws_eip.nat.id}"
```

Comment:
I don't think either of these references exists--there's no `aws_eip` called `nat`, and there's no `aws_subnet` called `public`. And we'll want to use *different* EIPs and subnets for each NAT gateway.

Reply:
yeah these were just copied from the example for now. we figured we'd tackled this better in the morning on dev without rushing into things.